node/coinstacks/solana/api/src/types.ts

```diff
@@ -1,3 +1,39 @@
+import { Logs, ParsedTransactionWithMeta } from '@solana/web3.js'
+
+export type Transaction = Omit<ParsedTransactionWithMeta, 'slot'>
+
+export interface WebsocketSubscribeResponse {
+  jsonrpc: '2.0'
+  method: string
+  result: number
```

Comment:
might make this result `unknown` to be more clear in that it can be a number or boolean in the case of subscribe/unsubscribe

Reply:
Smart, done in [37ba9ed](https://github.com/shapeshift/unchained/pull/1053/commits/37ba9edb1be37c7338ff96166dc24614031fbd65)